src/basic_recipes/poly.jl

```diff
@@ -163,16 +177,21 @@ function plot!(plot::Mesh{<: Tuple{<: AbstractVector{P}}}) where P <: Union{Abst
 
     attributes[:colormap] = get(plot, :colormap, nothing)
     attributes[:colorrange] = get(plot, :colorrange, nothing)
+    attributes[:lowclip] = get(plot, :lowclip, automatic)
+    attributes[:highclip] = get(plot, :highclip, automatic)
 
     bigmesh = if color_node[] isa AbstractVector && length(color_node[]) == length(meshes[])
         # One color per mesh
-        lift(meshes, color_node, attributes.colormap, attributes.colorrange) do meshes, colors, cmap, crange
+        lift(meshes, color_node, attributes.colormap, attributes.colorrange, attributes.lowclip, attributes.highclip) do meshes, colors, cmap, crange, lclip, hclip
             # Color are reals, so we need to transform it to colors first
             single_colors = if colors isa AbstractVector{<:Number}
                 interpolated_getindex.((to_colormap(cmap),), colors, (crange,))
             else
                 to_color.(colors)
             end
+            # Consider low- and highclips
+            single_colors[colors .< crange[1]] .= parse(RGBA, lclip)
+            single_colors[colors .> crange[2]] .= parse(RGBA, hclip)
```

Comment:
```suggestion
            single_colors[colors .< crange[1]] .= to_color(lclip)
            single_colors[colors .> crange[2]] .= to_color(hclip)
```

This handles symbols as well as RGB, HSV, and other color types.

Reply:
Thanks for the hint! I changed that accordingly.

Reply:
I tried that, but that does not work with the example I gave before. I could not really understand why, but doing these changes solved the problem. Do you have any idea?

Reply:
I think that the problem is that the colors are passed to `mesh()` already as colors and not as values (see [these lines](https://github.com/JuliaPlots/Makie.jl/blob/master/src/basic_recipes/poly.jl#L167-L186)). Therefore, it is not possible to determine whether a certain color should be clipped or not. This is done like this, because we want to have one color per polygon, which is different to the behavior of mesh, where one color per node is defined.

Reply:
Yeah, makes sense in that case.

Reply:
Ah damn, I was already in the PR https://github.com/JuliaPlots/Makie.jl/pull/2056, where I cleaned up the multi mesh recipe, to pass through numbers directly to the mesh call...
I'll open a PR with your changes and the changes from #2056, to do this more cleanly ;)

Reply:
That sounds good to me. I was thinking that such a solution would be cleaner, but then was struggling with passing numbers to mesh directly.

Reply:
https://github.com/JuliaPlots/Makie.jl/pull/2183